package.json

```diff
@@ -122,13 +122,13 @@
     "file-loader": "1.1.5",
     "install": "0.10.2",
     "json-loader": "0.5.7",
-    "minimist": "1.2.0",
     "node-sass": "4.7.2",
     "redux-logger": "^3.0.6",
     "sass-loader": "6.0.6",
     "style-loader": "0.19.0",
     "svg-inline-loader": "0.8.0",
     "url-loader": "0.6.2",
-    "webpack": "3.8.1"
+    "webpack": "3.8.1",
+    "webpack-merge": "^4.1.1"
```

Comment:
let's use exact versions in `package.json`

Reply:
A question about this: is that still necessary to use exact versions when we have a lock file?

Reply:
Hmm, fair point, but I'd still prefer exact ones. Up to you.